<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataMonitor</class>
 <widget class="QMainWindow" name="DataMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monitor transmisji</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButtonClear">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>26</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Wyczyść okno</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>118</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButtonFontMinus">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>Zmniejsz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFontDesc">
          <property name="text">
           <string>Aktualny rozmiar czcionki:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFontSize">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonFontPlus">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>Zwiększ</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>118</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonClose">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>26</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ukryj okno</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTextBrowser" name="textBrowserMonitor">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButtonClear</tabstop>
  <tabstop>pushButtonClose</tabstop>
  <tabstop>textBrowserMonitor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
